index.bs

```diff
@@ -669,16 +678,21 @@ authorizing an authenticator with which to complete the operation.
         credentials for the same account on a single authenticator. The platform is requested to return an error if the new
         credential would be created on an authenticator that also contains one of the credentials enumerated in this parameter.
 
+    - The <dfn>attachment</dfn> parameter contains authenticator attachment descriptions, which are used as an additional
+        constraint on which authenticators are eligible to participate in a [[#makeCredential]] or [[#getAssertion]] operation.
+        See [[#attachment]] for a description of the attachment values and their meanings.
+
+    - The <dfn>verificationGesture</dfn> parameter is intended for use by <a>[RPS]</a> that want authenticators to perform the 
+        <a>user verification</a> process during the [[#makeCredential]] operation. A gesture that can be 
+        performed by anyone in posession of an authenticator is not considered a gesture that verifies the user, such as 
```

Comment:
This caveat should be in the definition of "user verification" (both https://w3c.github.io/webauthn/#user-verification and the behavioral definition in https://w3c.github.io/webauthn/#authenticatormakecredential) rather than just at some call sites.